### PR TITLE
Morse Code Convertor using Vanilla JS #26

### PR DESCRIPTION
**Pull request by AftabMankapure**
# Pull Request for WebMasterLog 💡

## Issue Title: Morse Code Convertor using Vanilla JS

- **Name of the project** :  Morse code convertor 
- **Category:**  Vanilla-JS
- **Name:** Aftab Mankapure
- **GitHub ID:**  https://github.com/AftabMankapure
- **Idenitfy yourself:**  Contributor under JWOC

Closes:  #26 

### Describe the add-ons or changes you've made 📃

added  Morse Code Convertor using Vanilla JS #26 

## Type of change ☑️

Added new feature Morse Code Convertor using Vanilla JS 

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by AftabMankapure** [comment]
pls check this pr

File: Vanilla-JS-Projects/README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ To get started with these Vanilla JavaScript projects, follow these steps:
 |   9   | [Hex Color Code Generator](./Intermediate/Hex-color-code-generator-website) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |  10   | [Countdown Timer](./Intermediate/Countdown-Timer) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |  11   | [Recipe Finder](./Advanced/Recipe-Finder/) | ![Advanced](https://img.shields.io/badge/Advanced-FF0000?style=for-the-badge) |
+|  12   | [Morse Code Generator](./Advanced/Recipe-Finder/) | ![Advanced](https://img.shields.io/badge/Advanced-FF0000?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/README.md:59]: Set the path proper and linked with intermediate directory and add the label of Intermediate by removing Advanced.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Do these changes and again mentioned to me.

File: Vanilla-JS-Projects/README.md (hunk 1 above)
Comment (AftabMankapure) [reply Vanilla-JS-Projects/README.md]: check

**Comment by AftabMankapure** [comment]
check now

**Review by AftabMankapure — COMMENTED** [review]
done

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**